packages/@react-spectrum/actionbar/docs/ActionBar.mdx

```diff
@@ -0,0 +1,276 @@
+export default Layout;
+
+import Anatomy from './ActionBarAnatomy.svg';
+import docs from 'docs:@react-spectrum/actionbar';
+import {HeaderInfo, PropTable, PageDescription, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/actionbar/package.json';
+
+```jsx import
+import {ActionBar, ActionBarContainer, Item} from '@react-spectrum/actionbar';
+```
+
+---
+category: Toolbars
+keywords: [actionbar, selection, actions, collections, toolbar]
+after_version: 3.1.0
+---
+
+# ActionBar
+
+<PageDescription>{docs.exports.ActionBar.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ActionBar, ActionBarContainer, Item']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/action-bar/'}
+  ]} />
+
+## Example
+```tsx example
+function StaticActionBar() {
+  return (
+    <ActionBarContainer>
+      <ActionBar
+        isEmphasized
+        selectedItemCount={3}>
+        <Item key="edit">
+          Edit
+        </Item>
+        <Item key="copy">
+          Copy
+        </Item>
+        <Item key="delete">
+          Delete
+        </Item>
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+<StaticActionBar />
+```
+
+## Content
+<Anatomy />
+
+An ActionBar is a companion component intended to facilitate bulk actions on selected items within a collection component. It consists of an ActionGroup, a clear button, and a count of selected items. It accepts a list of <TypeLink links={docs.links} type={docs.exports.Item} />
+elements as children, each with a `key` prop. Alternatively, a function that returns `Item` elements is also supported as seen below. See the [collection components](../react-stately/collections.html)
+docs for more details about this API. These `Item` elements are used to populate the ActionBar's ActionGroup and thus follows the [same content guidelines](./ActionGroup.html#content) when it comes to customization and labeling.
+
+```tsx example
+function DynamicActionBar() {
+  let barItems = [
+    {key: 'edit', label: 'Edit'},
+    {key: 'copy', label: 'Copy'},
+    {key: 'delete', label: 'Delete'}
+  ];
+
+  return (
+    <ActionBarContainer>
+      <ActionBar
+        items={barItems}
+        selectedItemCount={3}
+        isEmphasized>
+        {item => (
+          <Item key={item.key}>
+            {item.label}
+          </Item>
+        )}
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+
+<DynamicActionBar />
+```
+
+### Layout
+ActionBarContainer accepts an ActionBar alongside any renderable children. It accepts the typical selection of [style props](#props), allowing you to customize its layout.
+The example below applies a static height on the ActionBarContainer instead of matching the height of the TableView.
+
+```tsx example
+import {Cell, Column, Row, TableBody, TableHeader, TableView} from '@react-spectrum/table';
+import {Text} from '@react-spectrum/text';
+
+function ActionBarTable(props) {
```

Comment:
Maybe this example should have ` onClearSelection` hooked up? While that functionality is demonstrated in the events section below, it feels like users may just see this example first and be confused as to why the buttons don't seem to do anything to selection.

Happy to go either way, bit torn on whether or not it is necessary

Reply:
Including `onClearSelection` makes sense. More so than hooking up the buttons which I was concerned about, since they also do nothing.

Reply:
Updated